mobilecoind-json/src/data_types.rs

```diff
@@ -1446,10 +1462,25 @@ mod test {
                 .pseudo_output_commitments,
             proto2.get_tx().get_signature().pseudo_output_commitments
         );
+        assert_eq!(
+            proto_proposal.get_tx().get_signature().range_proof_bytes,
+            proto2.get_tx().get_signature().range_proof_bytes,
+        );
         assert_eq!(
             proto_proposal.get_tx().get_signature().range_proofs,
             proto2.get_tx().get_signature().range_proofs
         );
+        assert_eq!(
+            proto_proposal
+                .get_tx()
+                .get_signature()
+                .pseudo_output_token_ids,
+            proto2.get_tx().get_signature().pseudo_output_token_ids,
+        );
+        assert_eq!(
+            proto_proposal.get_tx().get_signature().output_token_ids,
+            proto2.get_tx().get_signature().output_token_ids,
+        );
 
```

Comment:
Is there a follow-on ticket to implement mixed transactions tests for mobilecoind-json? I don't think it needs to be blocking, but maybe there's a ticket for "ensure mixed token types test coverage in all clients" that can have a prioritized checklist.

Reply:
I am not sure we're planning even to support multiple token types txs in the mobilecoind-json bindings -- let's sync with Eran, maybe I'm out of the loop

Reply:
I don't think you're out of the loop - just trying to figure out what our bookkeeping process is if we did decide to support it

Reply:
this is now #1867